README.md

```diff
@@ -40,6 +40,7 @@ framework when using Sauce Labs.
             - [WebdriverIO and Sauce Labs Front-End Performance](webdriverio/webdriver/examples/performance)
             - [WebdriverIO with Up and Downloading Files](webdriverio/webdriver/examples/up-download)
             - [WebdriverIO, Sauce Labs and W3C](webdriverio/webdriver/examples/w3c)
+            - [WebdriverIO with TypeScript](webdriverio/web/examples/typescript)
```

Comment:
You are adding a new folder called web here, this is not the structure of this repo. Can you please put everything under the folder `webdriver/examples/typescript`?

Reply:
@wswebcreation hey bud, this folder structure here is confusing. I don't understand the difference between `webdriverio/web/examples` and `webdriverio/webdriver/examples`.
![Screen Shot 2021-03-25 at 9 19 26 AM](https://user-images.githubusercontent.com/2136245/112479405-64509b80-8d4b-11eb-8d2d-b90064f58a34.png)

Reply:
What do you mean? There is are only 3 folders, did you merge with latest master?